tests/test_template_types.py

```diff
@@ -1178,6 +1182,41 @@ def test_is_message_empty_email_and_letter_templates(
     assert template.is_message_empty() == expected_result
 
 
+@pytest.mark.parametrize('template_class, template_type', (
+    (HTMLEmailTemplate, 'email'),
+    (LetterPrintTemplate, 'letter'),
+))
+@pytest.mark.parametrize('content, values', [
+    ("Some content", {}),
+    ("((placeholder)) some content", {"placeholder": ""}),
+    ("Some content ((placeholder))", {"placeholder": ""}),
+    pytest.param(
+        "((placeholder))", {"placeholder": "Some content"},
+        marks=pytest.mark.xfail(raises=AssertionError),
+    ),
+])
+def test_is_message_empty_email_and_letter_templates_tries_not_to_count_chars(
+    mocker,
+    template_class,
+    template_type,
+    content,
+    values,
+):
+    template = template_class({
+        "content": content, 'subject': 'Hi', 'template_type': template_type,
+    })
+    mock_content = mocker.patch.object(
+        template_class,
+        'content_count',
+        create=True,
+        new_callable=mock.PropertyMock,
+        return_value=None,
+    )
+    template.values = values
+    template.is_message_empty()
```

Comment:
Optional: Could assert that this is false as a sanity check that we've set up the rest of the test correctly